cli/src/lib/cli.ts

```diff
@@ -40,12 +66,7 @@ export const cli = async () => {
             default: "main",
           }),
       async (yargs) => {
-        await checkCommand(
-          yargs.ethscankey,
-          NetworkSchema.parse(yargs.network),
-          yargs.upgradeDirectory,
-          yargs.ref
-        );
+        await checkCommand(env, yargs.upgradeDirectory, yargs.ref);
```

Comment:
This is the pattern now. Just send the env object with all the configuration.

Reply:
Beautiful!